components/app_base/app/controllers/application_controller.rb

```diff
@@ -1,6 +1,6 @@
 class ApplicationController < ActionController::Base
   def current_user
-    raise 'Never call this method. Call current_user_identifier instead'
+    raise 'Never call this method. Call current_user instead'
```

Comment:
The message seems a bit odd.
Maybe we should raise `NotImplementedError`, or something similar?